lib/src/bloc/movies_bloc.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:async';
+
+import '../model/movie.dart';
+
+import '../resources/movie_repository.dart';
+
+class MoviesBloc {
+  final _repository = MovieRepository();
```

Comment:
you should rename this to make it more representative

Reply:
Fixed